examples/react-native/storybook/ui/InnerContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SafeAreaView, StyleSheet } from 'react-native';
+
+interface InnerContainerProps {
+  children: React.ReactNode;
+}
+
+export const InnerContainer = ({ children }: InnerContainerProps) => {
+  return <SafeAreaView style={[styles.container]}>{children}</SafeAreaView>;
```

Comment:
Curious why we didn't just use the `InnerContainer` component from `DefaultContainer` pkg. Is it because we need SafeAreaView here? Concerned that if we make changes to the InnerContainer we'll have to make changes here as well [either props or styling].

Reply:
Good point! I'll import it from the `DefaultContainer` pkg